fix(file-log) Soften performance warning (#2392)
Kong/docs.konghq.com: app/_hub/kong-inc/file-log/index.md

publisher: Kong Inc.
version: 1.0.0

desc: Append request and response data to a log file
description: |

>   Append request and response data to a log file (as JSON). You can also specify

Reviewer: Same comments as on prior version

>   Append request and response data to a log file (as JSON). You can also specify

Reviewer: ```suggestion
  Append request and response data in JSON format to a log file. You can also specify
```

>   streams, e.g. `/dev/stdout` and `/dev/stderr` - this is especially useful

Reviewer: ```suggestion
streams (for example, `/dev/stdout` and `/dev/stderr`), which is especially useful
```

  when running Kong in Kubernetes.

>   This plugin uses blocking I/O - this could affect performance when writing

Reviewer: ```suggestion
  This plugin uses blocking I/O, which could affect performance when writing
```

>   to physical files on slow - e.g. spinning - disks.

Reviewer: ```suggestion
  to physical files on slow (spinning) disks.
```
